src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !defined(CONFIG_INTEGER) || !defined(CONFIG_STRING) || !defined(CONFIG_METHODSET)
+#error CONFIG_INTEGER, CONFIG_STRING, and CONFIG_METHODSET must be defined before including this file.
+#endif // !defined(CONFIG_INTEGER) || !defined(CONFIG_STRING) || !defined(CONFIG_METHODSET)
+
+#if defined(DEBUG)
+CONFIG_INTEGER(AltJitLimit, W("AltJitLimit"), 0) // Max number of functions to use altjit for (decimal)
+CONFIG_INTEGER(AltJitSkipOnAssert, W("AltJitSkipOnAssert"), 0) // If AltJit hits an assert, fall back to the fallback JIT. Useful in conjunction with COMPLUS_ContinueOnAssert=1
```

Comment:
I think that the distinction that really matters here is supported knobs vs. unsupported knobs (which I assume is what INTERNAL/EXTERNAL was really trying to convey). At this point, I think that we have to assume that any knobs that are available in release builds are effectively supported. If that is the case (and please let me know if it is not), then I think that the existing partitioning of values between debug-only and not-debug-only is sufficient with the addition of a comment at the top of the file explaining the distinction. If we're willing to take another strategy, however, I'd be happy to add `INTERNAL`/`EXTERNAL` prefixes to the comments explaining each option. If we later find that we can use this information, we can move it into the macro at that time.